packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -636,10 +638,19 @@ export type ObjectFieldsConnection = {
   pageInfo: PageInfo;
 };
 
-/** Onboarding step */
-export enum OnboardingStep {
+/** Onboarding status */
+export enum OnboardingStatus {
```

Comment:
I feel we are mixing a lot of things here. I understand that it's good to have this computed on the BE but I would separate the subscriptionStatus and the workspaceStatus from here:
- workspaceStatus = ONGOING_CREATION | ACTIVE | INACTIVE | HYBERNATED
- subscriptionStatus = CANCELED | PAST_DUE | UNPAID | TRIALING
- onboardingStatus (userLevel actually) = PROFILE_CREATION | SUBSCRIPTION | SYNC_EMAIL | USER_CREATION